packages/app/src/app/overmind/effects/api/index.ts

```diff
@@ -0,0 +1,380 @@
+import apiFactory, { Api, ApiConfig } from './apiFactory';
+import {
+  CurrentUser,
+  Dependency,
+  Sandbox,
+  Module,
+  GitChanges,
+  EnvironmentVariable,
+  PopularSandboxes,
+  SandboxPick,
+  PickedSandboxes,
+  UploadedFilesInfo,
+  Directory,
+  GitInfo,
+  GitCommit,
+  GitPr,
+  RoomInfo,
+  PaymentDetails,
+  Profile,
+  UserSandbox,
+} from '@codesandbox/common/lib/types';
+import { TemplateType } from '@codesandbox/common/lib/templates';
+import { client } from 'app/graphql/client';
+import { LIST_TEMPLATES } from 'app/pages/Dashboard/queries';
+
+let api: Api;
+
+export default {
+  initialize(config: ApiConfig) {
```

Comment:
@christianalfoni I think it will be clearer what's exported if you declare the functions first and then export them in 1 go.

```ts
const initialize = (config: ApiConfig) => {
  // ...
};

const createPatronSubscription = (token: string, amount: number, coupon: string) => {
  // ...
};

// ...

export default {
  initialize,
  createPatronSubscription,
  // ...
}
```